src/network-config.js

```diff
@@ -0,0 +1,86 @@
+import {
+  getEnsRegistryAddress,
+  getFortmaticApiKey,
+  getPortisDappId,
+} from './local-settings'
+
+const localEnsRegistryAddress = getEnsRegistryAddress()
+const fortmaticApiKey = getFortmaticApiKey()
+const portisDappId = getPortisDappId()
+
+export const networkConfigs = {
+  main: {
+    addresses: {
+      ensRegistry:
+        localEnsRegistryAddress || '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
```

Comment:
We might want [to rely on Connect](https://github.com/aragon/connect/blob/0c542a44f862349d8843eb340c7e6600213d4aed/packages/connect-core/src/params.ts) for some of the information in the network configs, like the ENS registry.